<?xml version="1.0" encoding="UTF-8"?> 
<launch>
  <!-- 
    robot_to_mc_node.launch
    VERSION 0.0.0 Last Changed 2019-05-05
    Communicates over wifi to send/recv ros msgs
    If YEAR = 2019, then YR=19,
      robot (src) addr = 192.168.1.YR
      robot (src) port = 55YR
    MC: dst_addr = 192.168.1.10, dst_port = 5510
  -->
  <!-- Interface: 
    Pub:
      joy (sensor_msgs/Joy)

    Sub:

    Param:
      frequency (double) 100.0; The frequency of the read loop
  -->
  <node pkg="bigrdigr_control" type="robot_to_mc_node" name="robot_to_mc_node" output="screen" respawn="true">
    <param name="frequency" value="100.0" type="double"/>
    <param name="dst_addr" value="192.168.1.10" type="str"/>
    <param name="dst_port" value="5510" type="int"/>
    <param name="src_port" value="5519" type="int"/>
    <param name="isTeleopCtrl" value="true" type="bool"/>
  </node>
</launch>
